static/src/app/app.js

```diff
@@ -0,0 +1,37 @@
+import angular from 'angular';
+import uirouter from 'angular-ui-router';
+
+import routing from './app.routes';
+import library from './library/library';
+import login from './login/login';
+
+import Styles from '../style/sass/main.scss';
+
+let app = {
+    template: require('./app.html'),
+    controller: 'AppController',
+    controllerAs: 'app'
+};
+
+class AppController {
+    constructor($log) {
+        this.name = 'Raster Foundry';
```

Comment:
I think this needs to use @nginject (should probably be used in a number of spots I think). There is one use of `.inject` I think I saw somewhere, but as a team we prefer to use the annotation because the other way is really error prone.